stdlib/Profile/docs/src/index.md

```diff
@@ -15,3 +17,18 @@ Profile.retrieve
 Profile.callers
 Profile.clear_malloc_data
 ```
+
+## Memory profiling
+
+```@docs
+Profile.Allocs.@profile
+```
+
+The methods in `Profile.Allocs` are not exported and need to be called e.g. as `Profile.Allocs.fetch()`.
+
+```@docs
+Profile.Allocs.start
+Profile.Allocs.stop
+Profile.Allocs.clear
+Profile.Allocs.fetch
```

Comment:
```suggestion
Profile.Allocs.clear
Profile.Allocs.fetch
Profile.Allocs.start
Profile.Allocs.stop
```
I would reorder to put start,stop at the bottom, since they shouldn't need to be called directly by anyone.